packages/web-runtime/src/services/auth/authService.ts

```diff
@@ -298,10 +298,7 @@ export class AuthService {
     // TODO: create UserUnloadTask interface and allow registering unload-tasks in the authService
     await this.store.dispatch('runtime/auth/clearUserContext')
     await this.store.dispatch('resetUserState')
-    await Promise.all([
-      this.store.dispatch('clearDynamicNavItems'),
-      this.store.dispatch('hideModal')
-    ])
```

Comment:
What did this even do?

Reply:
Closing a potentially open modal I guess 😄